internal/app/init/pkg/system/services/ntpd.go

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// nolint: dupl,golint
+package services
+
+import (
+	"github.com/autonomy/talos/internal/app/init/pkg/ntpd"
+	"github.com/autonomy/talos/internal/app/init/pkg/system/conditions"
+	"github.com/autonomy/talos/internal/pkg/userdata"
+)
+
+// NTPd implements the Service interface. It serves as the concrete type with
+// the required methods.
+type NTPd struct {
+	client *ntpd.NTP
+}
+
+// ID implements the Service interface.
+func (t *NTPd) ID(data *userdata.UserData) string {
+	return "ntpd"
+}
+
+// PreFunc implements the Service interface.
+func (t *NTPd) PreFunc(data *userdata.UserData) error {
+	n := ntpd.New(data.Services.NTPd)
```

Comment:
Just curious, is there a reason we don't just let the `Daemon` do this?

Reply:
The main reason is I want to ensure on boot ( prior to starting k8s services ) that we have the proper time set and not wait for the first time.tick to trigger ( could be up to ~20min ). 
I'm also staging it for what I'm expecting a proper ntp implementation to do -- set the system time via https://golang.org/pkg/syscall/#Adjtimex. And I think there's a limitation of the starting time being too far out of sync. So doing the inital sync during startup should address this.

Reply:
SGTM